webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -644,6 +651,64 @@
                     </ul>
                 </div>
                 <!-- /correlation -->
+
+                {% if request.user.has_perm('crashstats.reprocess_crashes') %}
+                <div id="reprocess" class="ui-tabs-hide" data-crash-id="{{ report.uuid }}">
```

Comment:
It would make more sense to me to have that crash id in the form in a hidden input instead of in a data attribute. That would probably allow this to work without any JS. Progressive enhancement and stuff. ;-)